docs/getting-started/as-a-designer/README.md

```diff
@@ -13,7 +13,7 @@
 
 [cedar-user-support-link]: https://rei.slack.com/messages/CA58YCGN4
 
-<cdr-img alt="Welcome" :src="$withBase(`/getting-started-for-designers/figma-at-rei.png`)"/>
+<cdr-img alt="Image of text being edited in Figma" :src="$withBase(`/getting-started-for-designers/figma-at-rei.png`)"/>
```

Comment:
Thanks for clarifying, Rebecca. Since screen readers supply the "image of" text, I would update this to: Text being edited in Figma

Reply:
Ok Meg, I changed—thanks for that!